libs/moment-adapter/README.md

```diff
@@ -0,0 +1,15 @@
+# Fundamental Moment date time adapter implementation
+
+[![npm version](https://badge.fury.io/js/%40fundamental-ngx%2Fplatform.svg)](//www.npmjs.com/package/@fundamental-ngx/platform)
```

Comment:
Links are from platform package

Reply:
removed this part at all since I'm not sure what links should be here if any

Reply:
removed these links at all since I'm not sure what links should be used here if any

Reply:
@dimamarksman We could use the future link for the npm package, which would be `www.npmjs.com/package/@fundamental-ngx/moment-adapter`